src/python/pants/option/global_options.py

```diff
@@ -1083,6 +1086,22 @@ class BootstrapOptions:
             """
         ),
     )
+    sandboxer = BoolOption(
```

Comment:
Should the option name include "experimental" in its name? e.g. `experimental_sandboxer`?

Reply:
I don't think so? I think `experimental` was supposed to mean "this backend might not stick around" or "this API is more unstable than usual". But we know we want this feature, I assume?

Reply:
I think it'd be reasonable to communicate "this may not work as expected, yet". The user-facing API won't change (since it's just "implementation details"), but we can use this to set expectations about reliability, potentially.

For this particular one, seems less necessary, though, since this is an optional feature for now, i.e. no warnings for `sandboxer = false`.